vsm

```diff
@@ -910,8 +926,14 @@ if __name__ == "__main__":
     parser.set_defaults(log_condition_checks=True)
     parser.add_argument('--log-format', choices=['catapult'],
                         help='Write log file in specified format')
+    parser.add_argument('--signal-number-file', type=str,
+            help='A .vsi file which maps all signal names to numbers',
```

Comment:
Fixed this and the other indentation inconsistencies. I was just using vim's default indentation which I misunderstood to be the same as the Python style guide.